src/content/docs/es/guides/deploy/deno.mdx

```diff
@@ -170,3 +170,5 @@ Si tu proyecto está almacenado en GitHub, la [página web de Deno Deploy](https
     ```
 
 📚 Lee más sobre [SSR en Astro](/es/guides/server-side-rendering/).
+
+[Adaptador Deno]: https://github.com/denoland/deno-astro-adapter
```

Comment:
This is a link shortcut, so we shouldn't translate this, as this breaks the links that reference `Deno adapter` above on the page.
```suggestion
[Deno adapter]: https://github.com/denoland/deno-astro-adapter
```